example_test.py

```diff
@@ -72,12 +73,6 @@ def test_create_file(self):
 
     def test_delete_file(self):
         """Test example.delete_file()
-
-        `self.fs.CreateFile()` is convenient because it automatically creates
-        directories in the fake file system and allows you to specify the file
-        contents.
-
-        You could also use `open()` or `file()`.
```

Comment:
Yes, that was the reason ;) Actually I just wanted to move copy-pasted code up, but did not do a good job of that.